Update the TinaDocs for deployment on Alilibab Cloud Server (#2892)
tinacms/tina.io: content/docs/tina-cloud/deployment-options/alibaba-cloud.mdx

```
ssh -keygen -t ed25519 -C "deploy-key"
# Press Enter to use the default path
# Leave the passphrase empty by pressing Enter
```

#### Add Public Key to Authorized Keys

Add the generated public key to the authorized\_keys file to allow GitHub Actions to authenticate:

```
cat ~/.ssh/id_ed25519.pub >> ~/.ssh/authorized_keys
```

## Configuring GitHub Secrets

After generating your SSH key, you'll need to add several secrets to your GitHub repository to enable secure deployment.

### Navigate to GitHub Secrets

1. GitHub repository → Settings
2. Secrets and variables → Actions
3. Click "New repository secret"

### Required GitHub Secrets

You'll need to configure the following secrets:

1. `SSH_PRIVATE_KEY` : The private key generated on your server
   * View it using: `cat ~/.ssh/id_ed25519`
   * Make sure to include the entire key, including the `-----BEGIN OPENSSH PRIVATE KEY----` and `-----END OPENSSH PRIVATE KEY-----`
2. `SERVER_HOST`: Your Alibaba Cloud server's IP address
   * This is the public IP address of your ECS instance
3. `SERVER_USERNAME`: The username for connecting to your server
   * Typically `root` or the user account you created on your ECS instance

> 4. `NEXT_PUBLIC_TINA_BRANCH`: The branch that Tina CMS should use for content

Reviewer: ```suggestion
4. `NEXT_PUBLIC_TINA_BRANCH`: The branch that TinaCMS should use for content
```

   * Usually your main branch (e.g., `main`  or `master`)

> 5. `NEXT_PUBLIC_TINA_CLIENT_ID`: Your Tina Cloud client ID

Reviewer: ```suggestion
5. `NEXT_PUBLIC_TINA_CLIENT_ID`: Your TinaCloud client ID
```

>    * Find this in your Tina Cloud dashboard under project settings

Reviewer: ```suggestion
   * Find this in your TinaCloud dashboard under project settings
```

> 6. `TINA_TOKEN`: Your Tina Cloud token for authentication

Reviewer: ```suggestion
6. `TINA_TOKEN`: Your TinaCloud token for authentication
```

>    * Generate this from your Tina Cloud dashboard under API Access

Reviewer: ```suggestion
   * Generate this from your TinaCloud dashboard under API Access
```
